qa/integration-tests-engine/src/test/resources/org/camunda/bpm/integrationtest/functional/spin/feel/feel-spin-xml-decision.dmn

```diff
@@ -2,6 +2,7 @@
 <definitions xmlns="http://www.omg.org/spec/DMN/20151101/dmn.xsd"
              id="definitions"
              name="camunda"
+             xmlns:camunda="http://camunda.org/schema/1.0/dmn"
```

Comment:
Added `xmlns:camunda="http://camunda.org/schema/1.0/dmn"` as it is the required namespace for `historyTimeToLive`